test/e2e/bindata/assets/03_clusterrole.yaml

```diff
@@ -38,20 +40,30 @@ rules:
       - list
       - create
       - update
+      - delete
+      - patch
   - apiGroups:
       - rbac.authorization.k8s.io
     resources:
       - clusterroles
       - clusterrolebindings
     verbs:
-      - '*'
+      - create
+      - delete
+      - deletecollection
+      - get
+      - list
+      - patch
+      - update
+      - watch
   - apiGroups:
       - apps
     resources:
       - deployments
```

Comment:
I forgot to add `replicasets` in my previous PR in here. Would you mind if you add ` - replicasets` in here. I know, this is not related to the purpose of this PR but this is a minor change?.